queries.sql

```diff
@@ -8,3 +8,54 @@ SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
 SELECT * FROM animals WHERE neutered = TRUE;
 SELECT * FROM animals WHERE name <> 'Gabumon';
 SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+/*1*/
+
+BEGIN;
+UPDATE animals SET species = 'unspecified';
+SELECT * FROM animals;
```

Comment:
👍 Good work on working with transactions! However, it seems that the transaction rollback might not be functioning as expected. Here's a recommendation:

- [ ] Review and update your SQL query for the transaction to ensure that it rolls back the changes correctly. Inside the transaction, make sure to update the `animals` table by setting the `species` column to "unspecified," verify that the change was made, and then perform a rollback. After the rollback, verify that the `species` column returns to its state before the transaction. Add the screenshot in the PR too.

By addressing this issue, you can ensure that the transaction behaves as required by the project's specifications. Keep up the good work! 👍🔀🐾

Reply:
![image](https://github.com/lerfast/vet_clinic_database/assets/111823606/75c8ff6f-81f4-4ca2-abf8-cec2032884d7)
![image](https://github.com/lerfast/vet_clinic_database/assets/111823606/49505aae-a3dd-4885-9be6-738da0d007a2)